Formula/d/djview4.rb

```diff
@@ -52,34 +46,22 @@ def install
     # NOTE: Do not use command "make install".
     # Simply copy the application bundle where you want it.
     if OS.mac?
-      prefix.install "src/djview.app"
-      bin.write_exec_script prefix/"djview.app/Contents/MacOS/djview"
+      # This script creates a more complete bundle, still relying on homebrew libs
+      cd "mac" do system "sh", "./make_djview_bundle_small.sh" end
```

Comment:
If you want to use `do` ... `end`, you need

```suggestion
      cd "mac" do
        system "sh", "./make_djview_bundle_small.sh"
      end
```
Otherwise you can use braces as in my original suggestion:

```suggestion
      cd "mac" { system "sh", "./make_djview_bundle_small.sh" }
```